### DISCUSSION TITLE
How to remove flet logo before app loading

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#99] [Q&A]
Before loading the page of a web app, the flet logo appears, how to change or even remove the animation?

**Comment by FeodorFitsner** [comment]
I'm going to write a separate article next, but anyway, here you go. Animation displays `/icons/icon-192.png`, but it's embedded into Flet server executable. However, if you configure "assets" directory for your app as described here: https://flet.dev/docs/controls/image#src you will be able to override that image by putting yours inside `assets` folder with exactly same name:

```
/assets
   /icons
      /icon-192.png
main.py
```

Using this technique you can override any image or favicon used by Flet. Just notice in browser developer tools which URL it is requesting.

Let me know if you have any questions.

**Reply by nascin** [reply]
I followed the steps above and still couldn't change it, even following the blog post.

**Reply by FeodorFitsner** [reply]
Will do an example shortly!

**Reply by nascin** [reply]
Thanks

**Reply by FeodorFitsner** [reply]
https://github.com/flet-dev/examples/tree/markdown-animations/python/apps/custom-loading-image

**Reply by jet10000** [reply]
https://github.com/flet-dev/examples/tree/main/python/apps/custom-loading-image

**Reply by jet10000** [reply]
I want to customize the title of the index.html page, but the $FLUTTER_BASE_HREF parameter in the page cannot be obtained and can only be manually set to "/"

<img width="193" alt="image" src="https://user-images.githubusercontent.com/2258120/191812092-636a017d-67ca-4dc2-a6e0-d266b88ee21a.png">

**Reply by FeodorFitsner** [reply]
To change page title:

```python
page.title = "Your title"
page.update()
```

**Reply by jet10000** [reply]
`page.title = "Your title"` is only displayed after the app is loaded, and "flet" is displayed during the app loading, so in order to customize the information during the app loading, the index.html page must be customized, but the $FLUTTER_BASE_HREF parameter in the page is invalid.

<img width="479" alt="image" src="https://user-images.githubusercontent.com/2258120/191816206-b805718f-d7e7-4096-8e50-4f5b9e0bcfce.png">

https://github.com/flet-dev/flet/tree/main/client/web

**Reply by FeodorFitsner** [reply]
I see. In your own `index.html` it should be always `<base href="/">`.
